src/lib/lore.sh

```diff
@@ -160,57 +160,147 @@ function is_introduction_patch()
   return 1
 }
 
-# Verify if the target URL is accessible or not.
+# This function extracts the patch tag of a lore message title. A patch tag is a
+# string surrounded by square brackets that contains the word "PATCH" and/or
+# "RFC".
 #
-# @url Target url
+# @message_title: Message title of a lore message.
 #
 # Return:
-# If the URL is accessible, return 0. Otherwise, return 22.
-function is_the_link_valid()
+# If the function finds a patch tag as a substring of `@message_title`, outputs
+# the patch tag, otherwise, output an empty string.
+function get_patch_tag()
 {
-  local url="$1"
-  local curl_cmd='curl --insecure --silent --fail --silent --head'
-  local raw_curl_output
-  local url_status_code
+  local message_title="$1"
+  local patch_tag
 
-  [[ -z "$url" ]] && return 22 # EINVAL
+  if [[ "$message_title" =~ \[[^\]]*(RFC|Rfc|rfc|PATCH|Patch|patch)[^\[]*\] ]]; then
+    patch_tag="${BASH_REMATCH[0]}"
+  fi
```

Comment:
In case there is no match, I implicitly determined that the default tag is an empty string. The rest of the `get_<patch_attribute>` handle an empty patch tag as "undefined"; for this situation, the version, number in series, and the total will be `X`. Do you agree with this approach?